indexing-service/src/main/java/org/apache/druid/indexing/common/KillTaskReport.java

```diff
@@ -65,17 +65,20 @@ public Object getPayload()
   public static class Stats
   {
     private final int numSegmentsKilled;
+    private final int numSegmentsKilledInDeepStorage;
     private final int numBatchesProcessed;
     private final Integer numSegmentsMarkedAsUnused;
 
     @JsonCreator
     public Stats(
         @JsonProperty("numSegmentsKilled") int numSegmentsKilled,
+        @JsonProperty("numSegmentsKilledInDeepStorage") int numSegmentsKilledInDeepStorage,
```

Comment:
Why add this property too? This is a user-facing property.

Reply:
I had added it to test the changes. I can remove it if it is unnecessary in the report

Reply:
lets remove it please. we will add it later if its needed.

Reply:
Done